### DISCUSSION TITLE
How do I build a debian package for i386?

### DISCUSSION THREAD

**Discussion by narc-Ontakac2** [FreeTubeApp/FreeTube#4944] [Q&A]
I am aware that this project uses makedeb for this. However I can't find a PKGBUILD file. In addition I would like to know if there is any cross building support (there are arm packages, so you might know how to do this with makedeb).

**Comment by absidue** [comment]
The makedeb version is unofficial (which versions are official and which ones aren't are listed in the README), please reach out to the maintainers of the makedeb stuff if you have questions specific to that.
